### DISCUSSION TITLE
How do i change the keybind for the app launcher

### DISCUSSION THREAD

**Discussion by Quills245** [end-4/dots-hyprland#728] [Q&A] [answered]
I don't like to have stuff set to just the super key, so i wanted to change the app launcher from oppening by just pressing that, but changing the binds from line 19 of the keybinds.conf file (`bind = , Super, exec, true # Open app launcher`) does nothing, in fact, i can even remove that line and nothing changes, the launcher still comes up, how do i change that, and what does this bind really do?

**Comment by Quills245** [comment, marked as answer]
Oh, managed to find out how it works, the thing i wanted to change was actually this "overview" one at line 155, yay

I still wonder what line 19 actually does tho, it isn't illustrative right? It would be a comment if it was

**Reply by end-4** [reply]
since opening the app launcher or terminal is the first thing a newcomer would do when they log in, keybinds for those actions are placed at the beginning to be the first ones shown on the cheatsheet (Super+/) so they'll find it right away

**Reply by end-4** [reply]
they're just there so the cheatsheet's keybind parser finds it and puts in the list
`true` does nothing
the real thing is at line 155 like you mentioned
